proseco/fid.py

```diff
@@ -278,16 +278,28 @@ def get_fid_candidates(self):
         # columns.
         cand_fids = FidTable([ids, yang, zang, row, col],
                              names=['id', 'yang', 'zang', 'row', 'col'])
-        self.log(f'Initial candidate fid ids are {cand_fids["id"].tolist()}')
-
-        # Reject (entirely) candidates that are off CCD or have a bad pixel
-        self.reject_off_ccd(cand_fids)
-        self.reject_hot_or_bad_pixel(cand_fids)
-
         shape = (len(cand_fids),)
         cand_fids['mag'] = np.full(shape, FID.fid_mag)  # 7.000
         cand_fids['spoilers'] = np.full(shape, None)  # Filled in with Table of spoilers
         cand_fids['spoiler_score'] = np.full(shape, 0, dtype=np.int64)
+
+        self.log(f'Initial candidate fid ids are {cand_fids["id"].tolist()}')
+
+        # Reject candidates that are off CCD, have a bad pixel, or are spoiled
+        # Check for spoilers only against stars that are bright enough and on CCD
+        # (within dither).
+        idx_bads = []
+        stars_mask = ((self.stars['mag'] < FID.fid_mag - ACQ.col_spoiler_mag_diff) &
+                      (np.abs(self.stars['row']) < 512 + self.dither_guide.row))
```

Comment:
I also imagine that the error could be larger than dither_guide_row at the beginning of an observation, but we're not going to find stars bright enough to spoil a fid and just off the CCD enough for that to matter.

Reply:
Well the maneuver error is corrected quickly enough that for the bulk of the observation the thing that really matters is the guide dither.  If a fid is spoiled for the first 60 seconds that isn't a huge problem.

Reply:
I figured any issue for a fid and it was just done, if we don't re-search.